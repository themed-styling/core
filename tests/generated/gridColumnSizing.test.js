import 'jest-styled-components'

import { gridColumnSizing } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(gridColumnSizing, 'gridColumnSizing')
