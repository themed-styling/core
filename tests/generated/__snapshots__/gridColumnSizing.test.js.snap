// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gridColumnSizing renders 0 1`] = `
Array [
  .c0 {
  grid-column-sizing: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders function 1`] = `
Array [
  .c0 {
  grid-column-sizing: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: has;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: has;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders invalid theme path 1`] = `
Array [
  .c0 {
  grid-column-sizing: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders negative number 1`] = `
Array [
  .c0 {
  grid-column-sizing: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  grid-column-sizing: July;
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: July;
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-column-sizing: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-column-sizing: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  grid-column-sizing: July;
}

@media May {
  .c0 {
    grid-column-sizing: August;
  }
}

@media April {
  .c0 {
    grid-column-sizing: September;
  }
}

@media January {
  .c0 {
    grid-column-sizing: November;
  }
}

@media April {
  .c0 {
    grid-column-sizing: August;
  }
}

@media June {
  .c0 {
    grid-column-sizing: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: July;
}

@media May {
  .c0 {
    grid-column-sizing: August;
  }
}

@media April {
  .c0 {
    grid-column-sizing: September;
  }
}

@media January {
  .c0 {
    grid-column-sizing: November;
  }
}

@media April {
  .c0 {
    grid-column-sizing: August;
  }
}

@media June {
  .c0 {
    grid-column-sizing: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-column-sizing: November;
  }
}

@media April {
  .c0 {
    grid-column-sizing: January;
  }
}

@media January {
  .c0 {
    grid-column-sizing: December;
  }
}

@media April {
  .c0 {
    grid-column-sizing: April;
  }
}

@media June {
  .c0 {
    grid-column-sizing: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-column-sizing: November;
  }
}

@media April {
  .c0 {
    grid-column-sizing: January;
  }
}

@media January {
  .c0 {
    grid-column-sizing: December;
  }
}

@media April {
  .c0 {
    grid-column-sizing: April;
  }
}

@media June {
  .c0 {
    grid-column-sizing: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    grid-column-sizing: July;
  }
}

@media April {
  .c0 {
    grid-column-sizing: August;
  }
}

@media January {
  .c0 {
    grid-column-sizing: September;
  }
}

@media April {
  .c0 {
    grid-column-sizing: November;
  }
}

@media June {
  .c0 {
    grid-column-sizing: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    grid-column-sizing: July;
  }
}

@media April {
  .c0 {
    grid-column-sizing: August;
  }
}

@media January {
  .c0 {
    grid-column-sizing: September;
  }
}

@media April {
  .c0 {
    grid-column-sizing: November;
  }
}

@media June {
  .c0 {
    grid-column-sizing: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-column-sizing: January;
  }
}

@media April {
  .c0 {
    grid-column-sizing: December;
  }
}

@media January {
  .c0 {
    grid-column-sizing: April;
  }
}

@media April {
  .c0 {
    grid-column-sizing: May;
  }
}

@media June {
  .c0 {
    grid-column-sizing: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-column-sizing: January;
  }
}

@media April {
  .c0 {
    grid-column-sizing: December;
  }
}

@media January {
  .c0 {
    grid-column-sizing: April;
  }
}

@media April {
  .c0 {
    grid-column-sizing: May;
  }
}

@media June {
  .c0 {
    grid-column-sizing: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders postitive number 1`] = `
Array [
  .c0 {
  grid-column-sizing: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders string 1`] = `
Array [
  .c0 {
  grid-column-sizing: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: June;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: June;
}

<div
    className="c0"
  />,
]
`;

exports[`gridColumnSizing renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`gridColumnSizing renders valid theme path 1`] = `
Array [
  .c0 {
  grid-column-sizing: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-column-sizing: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
